container_files/assets/www/doc/builtin/procedures/EMProcedure.md

```diff
@@ -0,0 +1,40 @@
+# EM Training Procedure
```

Comment:
Again, call this gaussian clustering or similar, not EM.  I don't think that EM should be used anywhere except in the description of the algorithm in the Training section.